source/_docs/z-wave/device-specific.markdown

```diff
@@ -50,6 +50,8 @@ echo -e -n "...turn on/off string from examples above..." | cu -l /dev/zstick -s
 
 ### Razberry Board
 
+GPIO is only supported on 32-bit version of Home Assistant OS.
```

Comment:
This is no longer true.

Reply:
I based this comment on https://www.home-assistant.io/hassio/installation/
I thought it was relevant to add the info here as this is a GPIO board. If this is no longer the case we should update the documentation in both places.

Reply:
I have reverted this.